.github/workflows/docker-images-nightly.yml

```diff
@@ -78,14 +73,15 @@ jobs:
         uses: docker/build-push-action@v5.1.0
         with:
           context: .
+          file: ./docker/images/n8n-custom/Dockerfile
           build-args: |
             N8N_RELEASE_TYPE=nightly
-          file: ./docker/images/n8n-custom/Dockerfile
-          platforms: ${{ github.event.inputs.include-arm64 == 'true' && 'linux/amd64,linux/arm64' || 'linux/amd64' }}
+          platforms: linux/amd64,linux/arm64
           provenance: false
           push: true
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
```

Comment:
I assume there' some mechanism to bust the cache that is happening?

Reply:
We have a cache size limit, and github deletes older caches automatically.